connector/src/main/resources/config.yml

```diff
@@ -38,6 +38,8 @@ remote:
   # The IP address of the remote (Java Edition) server
   # If it is "auto", for standalone version the remote address will be set to 127.0.0.1,
   # for plugin versions, Geyser will attempt to find the best address to connect to.
+  # By keeping it as auto, the address, port, and Floodgate support will be automatically configured.
+  # In standalone, keeping this as auto sets the remote address to 127.0.0.1 therefore Floodgate will not be Automatically configurated.
```

Comment:
```suggestion
  # In standalone, keeping this as auto sets the remote address to 127.0.0.1, and Floodgate will not be automatically configured.
```

Reply:
I still feel like it's the wrong conjunction word. Either way, it needs a comma.